src/picongpu/include/fields/FieldTmp.hpp

```diff
@@ -88,10 +86,14 @@ namespace picongpu
 
         static std::string getName();
 
-        uint32_t getCommTag();
-
         virtual EventTask asyncCommunication( EventTask serialEvent );
 
+        /** gather data from neighbor GPU
+         *
+         * copy data from the border of neighboring GPU into the local guard.
```

Comment:
Add missing information that there is no race condition if `asyncCommunicationGather` is called before or after `asyncCommunication`